openmmtools/multistate/multistatesampler.py

```diff
@@ -1606,14 +1636,57 @@ def _is_completed_static(iteration_limit, iteration, last_err_free_energy,
     @staticmethod
     def _throw_restoration_error(message):
         """Masking function to hide the RestorationError class without exposing it or making it a 'hidden' (_X) error"""
+
         class RestorationError(Exception):
             """Represent errors occurring during attempts to restore simulations."""
+
             def __init__(self, error_message):
                 super().__init__(error_message)
                 # Critical messages which have halted a simulation, badly
                 logger.critical(error_message)
+
         raise RestorationError(message)
 
+    @staticmethod
+    def _create_context_caches(energy_context_cache_input, propagation_context_cache_input):
```

Comment:
We can just call the arguments `energy_context_cache` and `propagation_context_cache`. We don't need the `_input` suffix.

Reply:
Fixed in latest changes.